src/zoning-district-class/zoning-district-class.repository.schema.ts

```diff
@@ -8,3 +11,10 @@ export type FindAllRepo = z.infer<typeof findAllRepoSchema>;
 export const findByIdRepoSchema = zoningDistrictClassEntitySchema;
 
 export type findByIdRepo = z.infer<typeof findByIdRepoSchema>;
+
+export const findCategoryColorsRepoSchema =
+  zoningDistrictClassCategoryColorsRepoSchema;
```

Comment:
Defining the ClassCategoryColor schema on-the-fly looks like: (Using [zod pick](https://zod.dev/?id=pickomit))

```suggestion
export const findCategoryColorsRepoSchema = z.array(zoningDistrictClassEntitySchema.pick({ category: true, color: true }));
```